services/QuillLMS/app/controllers/auth/clever_controller.rb

```diff
@@ -9,6 +9,15 @@ class Auth::CleverController < ApplicationController
   # since a teacher may have created more clever classrooms /students betwixt logins
   def clever
     auth_hash = request.env['omniauth.auth']
+    if session[:clever_redirect]&.include?('my_account')
+      user = current_user.update(email: auth_hash['info']['email'])
+      if user
+        session[:google_or_clever_just_set] = true
```

Comment:
Similarly, we'd want to constantizem`:clever_redirect` and  `:google_or_clever_just_set`, e.g. In `QuillAuthentication` (or somewhere appropriate), define `CLEVER_REDIRECT = :clever_redirect` and then always use that constant in code (so that typos will raise errors).